org.eclipse.scout.rt.ui.html/src/main/java/org/eclipse/scout/rt/ui/html/json/desktop/JsonOutline.java

```diff
@@ -53,6 +56,9 @@ public class JsonOutline<OUTLINE extends IOutline> extends JsonTree<OUTLINE> {
   private static final String PROP_SHOW_TILE_OVERVIEW = "showTileOverview";
   private static final String PROP_COMPACT_ROOT = "compactRoot";
 
+  public static final String EVENT_SEARCH_FILTER_FOR_PAGE_REQUEST = "searchFilterForPageRequest";
```

Comment:
TODO BSH: These events (and everthing connected to them) seem to be obsolete. Check and remove!